src/lib/babel-custom.js

```diff
@@ -64,6 +64,9 @@ export default babelPlugin.custom(babelCore => {
 						pragma: customOptions.jsx || 'h',
 						pragmaFrag: customOptions.jsxFragment || 'Fragment',
 					},
+					!customOptions.typescript && {
```

Comment:
I copied this from the previous Microbundle version, there we run the Flow plugin anytime it's not TypeScript.